site/en/blog/chrome-58-media-updates/index.md

```diff
@@ -0,0 +1,263 @@
+---
+# Required
+layout: 'layouts/blog-post.njk'
+
+# Required
+title: Audio/Video Updates in Chrome 58
+
+# Required
+description: >
+  A round up of the audio/video updates in Chrome 58.
+
+# Optional
+# How to add a new author
+# https://developer.chrome.com/docs/handbook/how-to/add-an-author/
+authors:
+  - beaufortfrancois
+
+# Required
+date: 2017-03-17
+
+# Optional
+# Include an updated date when you update your post
+updated: 2018-01-08
+
+# Optional
+# How to add a new tag
+# https://developer.chrome.com/docs/handbook/how-to/add-a-tag/
+tags:
+  - news
+  - media
+  - chrome58
+
+
+
+---
+
+
+- Developers can now [customize media controls](#controlslist) such as the
+  download, fullscreen and remoteplayback buttons.
+- Sites installed using the "Add to Homescreen" flow can [autoplay audio
+  and video in the manifest's scope](#autoplay).
+- Chrome on Android now [pauses autoplaying a muted video when it is invisible](#offscreen).
+- Developers can now access the approximate range of colors supported by Chrome and
+  output devices using the [`color-gamut` Media Query](#colorgamut).
+- When using Media Source Extensions, it's now possible to
+  [switch between encrypted and clear streams].
+
+## Media controls customization
+
+Developers can now customize Chrome's native media controls such as the
+download, fullscreen and [remoteplayback] buttons using the new [ControlsList API].
+
+<div class="float-right">
+<figure>
+{% Img src="image/T4FyVKpzu4WKF1kBNvXepbi08t52/lxOu6rBvfaDJrn2mk8bQ.png", alt="Native media controls in Chrome 58", width="800", height="507" %}
+  <figcaption>
+    <b>Figure 1.</b>
+    Native media controls in Chrome 58
+  </figcaption>
+</figure>
+</div>
+
+This API offers a way to show or hide native media controls that do not make
+sense or are not part of the expected user experience, or only allow a
+limited set of features.
+
+The current implementation for now is blocklist mechanism on native controls
+with the ability to set them directly from HTML content using the new
+attribute `controlsList`. Check out the [official
+sample](https://googlechrome.github.io/samples/media/controlslist.html).
+
+Usage in HTML:
+
+```html
+<video controls <b>controlsList="nofullscreen nodownload noremoteplayback"</b>></video>
+```
+
+Usage in JavaScript:
+
+```js
+    var video = document.querySelector('video');
+    video.controls; // true
+    video.controlsList; // ["nofullscreen", "nodownload", "noremoteplayback"]
+    video.controlsList.remove('noremoteplayback');
+    video.controlsList; // ["nofullscreen", "nodownload"]
+    video.getAttribute('controlsList'); // "nofullscreen nodownload"
+
+    video.controlsList.supports('foo'); // false
+    video.controlsList.supports('noremoteplayback'); // true
+```
```

Comment:
```suggestion
var video = document.querySelector('video');
video.controls; // true
video.controlsList; // ["nofullscreen", "nodownload", "noremoteplayback"]
video.controlsList.remove('noremoteplayback');
video.controlsList; // ["nofullscreen", "nodownload"]
video.getAttribute('controlsList'); // "nofullscreen nodownload"

video.controlsList.supports('foo'); // false
video.controlsList.supports('noremoteplayback'); // true
```

Reply:
I rarely review twice and this one was my fault. You also need to remove the indentation. Specifically, only remove the indented characters that are common to all lines. You'll see what I mean in the next comment.